action.yml

```
@@ -67,57 +67,57 @@ runs:
         if [[ -n "${{ inputs.config }}" ]]
         then
             echo "using config from '${{ inputs.config }}'"
-            CONFIG_PATH="--config ${{ inputs.config }}"
+            CONFIG_PATH=("--config" "${{ inputs.config }}")
```

```suggestion
            # Use arrays so that inputs can contain spaces
            CONFIG_PATH=("--config" "${{ inputs.config }}")
```
Do you think this comment explains what we are doing?
Feel free to expand this comment. 👍